linkerd/app/src/env.rs

```diff
@@ -189,8 +192,11 @@ const DEFAULT_INITIAL_CONNECTION_WINDOW_SIZE: u32 = 1048576; // 1MB ~ 16 streams
 // 10_000 is arbitrarily chosen for now...
 const DEFAULT_BUFFER_CAPACITY: usize = 10_000;
 
-const DEFAULT_INBOUND_ROUTER_MAX_IDLE_AGE: Duration = Duration::from_secs(60);
-const DEFAULT_OUTBOUND_ROUTER_MAX_IDLE_AGE: Duration = Duration::from_secs(60);
+const DEFAULT_INBOUND_ROUTER_MAX_IDLE_AGE: Duration = Duration::from_secs(10);
+const DEFAULT_OUTBOUND_ROUTER_MAX_IDLE_AGE: Duration = Duration::from_secs(3);
+
+const DEFAULT_INBOUND_MAX_IDLE_PER_ENDPOINT: usize = std::usize::MAX;
+const DEFAULT_OUTBOUND_MAX_IDLE_PER_ENDPOINT: usize = std::usize::MAX;
```

Comment:
The PR description suggests that only two idle connections will be kept in the pool. This suggests that we don't cap the number of idle connections by default. Do we expect the control plane to always set this to 2, or something? Or should we change the default value here?

Reply:
Update: ah, I see, this was explained in https://github.com/linkerd/linkerd2/issues/5146#issuecomment-723204412. Let's just be sure to update the commit message when merging?